CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Changes
+
+* [#591](https://github.com/rubocop/rubocop-rails/issues/591): Add `change_column` to `Rails/ReversibleMigration`. ([@mattmccormick][])
```

Comment:
Can you use the following changelog entry format style instead of this?

> Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop-rails/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

Reply:
Sorry, I missed that

Reply:
Updated